packages/core/src/controllers/ApiController.ts

```diff
@@ -60,8 +60,8 @@ export const ApiController = {
 
     return {
       'x-project-id': projectId,
-      'x-sdk-type': sdkType,
-      'x-sdk-version': sdkVersion
+      'x-sdk-type': sdkType || 'w3m',
```

Comment:
Handle this better way. How will be the SDK Type and SDK version handled in AppKit?

Reply:
Let's handle this changes in `feat/multi-chain`